From adcc1df0049e0093cb94c867bd2be8c9fe242a61 Mon Sep 17 00:00:00 2001
From: Peter Rabbitson <ribasushi@cpan.org>
Date: Tue, 13 Sep 2016 17:15:48 +0200
Subject: [PATCH] Fix for upcoming (not yet available via DBD::SQLite)
 libsqlite version
 .
 - Fix missing ORDER BY leading to failures of t/prefetch/grouped.t
   under upcoming libsqlite (RT#117271)
Bug-Debian: https://bugs.debian.org/835731
Bug: https://rt.cpan.org/Ticket/Display.html?id=117271


--- a/t/prefetch/grouped.t
+++ b/t/prefetch/grouped.t
@@ -100,7 +100,7 @@
 
   # add an extra track to one of the cds, and then make sure we can get it on top
   # (check if limit works)
-  my $top_cd = $cd_rs->slice (1,1)->next;
+  my $top_cd = $cd_rs->search({}, { order_by => 'cdid' })->slice (1,1)->next;
   $top_cd->create_related ('tracks', {
     title => 'over the top',
   });
